eks-anywhere-common/Addons/Partner/SUSE/NeuVector/testJob.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1
```

Comment:
This file should be moved to `https://github.com/aws-samples/eks-anywhere-addons/tree/main/eks-anywhere-common/Testers` with a new folder for SUSE. This is the location to place the tester job.